modules/logging_config/log.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# TODO: idk what the channel names are
+CHANNELS=(can_primary can_secondary)
```

Comment:
These are right. fyi you can see interface names and overlays in the hw_config, which is also in this repo.